### ISSUE TITLE
UI/UX - Toggle window and select the right tab when clicking UI icons

### ISSUE THREAD

**Issue by Zenrac** [mon/0x40-web#72]
**Issue description**
- When we click on the cog icon, we can click again to close the window, that's not the case for the question mark icon.
- When we click on the question mark icon, the info tab is displayed, when we click on the cog icon, the info tab is still displayed, nothing change.

**IMO: Expected missing behaviour**
- When clicking the question mark icon, it should close the window if already shown and already on the INFO tab (call the window toggle)
![image](https://user-images.githubusercontent.com/19573589/181024968-f4bc47ab-9e4c-4206-852d-d4c7338674b3.png)

- When clicking the cog icon, it should call `selectTab("OPTIONS")` to display the right tab if the window is closed or not on the OPTIONS tab (and keep the toggle window)
![image](https://user-images.githubusercontent.com/19573589/181025435-dbd89806-f82a-4006-966f-ace57cda2bbd.png)

**TL;DR:** Both buttons (cog and question mark) should share the same features (If window already displayed: select the right tab OR close the window if already on it. If window is not displayed: open the window and select the right tab)
